prog/vm/nexus.rb

```diff
@@ -291,6 +292,7 @@ def write_params_json
 
   label def run
     host.sshable.cmd("sudo systemctl start #{q_vm}")
+    vm.update(tainted_at: Time.now) unless vm.pool_id
```

Comment:
You probably need this in the prior label, since you have to consider that `start` runs but the database cannot commit. At that point, I presume in principle irreplacable/sensitive user data may be present.

Reply:
...or perhaps you don't.  I guess pooled stuff is special. But fair question if that's the case: could tainting happen even later?

Reply:
I moved it to the previous label.